support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerListener.java

```diff
@@ -8,16 +8,12 @@
 
 import javax.annotation.Nonnull;
 
-/**
- * @author Vivek Pandey
```

Comment:
Ha! No, I just looked at this class and realized that essentially nothing of what he originally wrote remains unedited, so leaving the attribution there is just misleading. If you want to know what someone wrote, check Git history.